py/selenium/webdriver/common/timeouts.py

```diff
@@ -35,55 +55,55 @@ def __init__(self, implicit_wait=0, page_load=0, script=0):
         self._script = self._convert(script)
 
     @property
-    def implicit_wait(self):
+    def implicit_wait(self) -> float:
```

Comment:
@AutomatedTester unsure whether `float`s are meant to be here. To me, it looks like the timeouts should be all integers (or the explicit conversion in `Timeouts._convert` is useless), but in Python 3, the division makes them to floats. Maybe the integer division `//` should be used here instead and was forgotten when dropping Python 2 support? I may be wrong, though, thus typed based on the current implementation.

Reply:
@AutomatedTester actually, disregard that, I see now that it's just the rounding.